dask/dataframe/tests/test_dataframe.py

```diff
@@ -3994,6 +3994,14 @@ def test_to_datetime():
         check_divisions=False,
     )
 
+    with pytest.raises(NotImplementedError) as exc:
+        dd.to_datetime("2021-08-03")
+    assert "non-index-able arguments" in str(exc.value)
```

Comment:
It might be cleaner to use `pytest.raises(... match="non-index-able arguments")

Reply:
sounds good, made the adjustment